modules/nf-core/nanomonsv/parse/main.nf

```diff
@@ -0,0 +1,43 @@
+process NANOMONSV_PARSE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::nanomonsv=0.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanomonsv:0.5.0--pyhdfd78af_0':
+        'quay.io/biocontainers/nanomonsv:0.5.0--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
```

Comment:
Is there CRAM support for this tool?

Reply:
The GitHub refers specifically to BAM (actually specifically to one aligned by minimap2). 
Internally it seems to use `pysam.AlignmentFile` with `rb` hardcoded so I am fairly certain it is restricted to BAM. If CRAM works it would be by chance I think.

Reply:
Allright LGTM then :)